src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * Duke bot object class, which has a task list, a storage space and a user interface.
+ */
+public class Duke {
+
+    private Ui ui;
+    private Storage storage;
+    private TaskList tasks;

```

Comment:
Should this be better named as a singular because TaskList is technically not a collection of tasks like ArrayList<Task>?

Reply:
Thanks for highlighting that

Reply:
I see, thanks for the suggestion!